### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by klikooo**
Changed variables names with less than 3 chars, also updated the indentation.

User story: As a developer I want to have understable code so that it is easier to understand and make changes to the code.

### CODE REVIEW

File: src/DUWOagent/DUWO.mod2g [hunk 1]
```diff
@@ -13,18 +13,18 @@ module DUWO {
...
 	% Demolish when we have this as a goal
 	if goal(goalDemolish) 
-		then demolishMod.
+			then demolishMod.
 	
 	% Build student housing when have it as goal.
 	% Use the normal module when we have enough budget
-	if not(bel(lowBudget)), not(bel(noBudget))
-		then normalMod.
+	if goal(goalBuildStudentHousing)
```
Comment (Denpeer) [inline src/DUWOagent/DUWO.mod2g]: goalReachBudgetTarget is indeed a goal when nobudget is true, but buildstudenthousing can be a goal regardless of budget

Comment (klikooo) [inline src/DUWOagent/DUWO.mod2g]: Aah yes you are right!

**Comment by Denpeer** [comment]
Tested and it works, Approved

File: src/DUWOagent/lowBudgetMod.mod2g [hunk 2]
```diff
@@ -11,10 +11,10 @@ exit=always.
 module lowBudgetMod {
 	
 	% Build student housing when have it as goal.
-	if bel(functions(FS), 
-		member([Name,Id,L],FS),
-		member('STUDENT',L),
+	if bel(functions(FunctionsList), 
+		member([Name,BuildingID,CatergorieList],FunctionsList),
```
Comment (MaxGroenenboom) [inline src/DUWOagent/lowBudgetMod.mod2g]: `CatergorieList` should be `CategoryList` or `CategoriesList`

**Comment by MaxGroenenboom** [comment]
Apart from the comment, I approve.

**Comment by Denpeer** [comment]
Goals aren't handled right anymore. They should be adopted in `goalEvents.mod2g`

File: src/DUWOagent/DUWO.mod2g [hunk 3]
```diff
@@ -17,14 +17,14 @@ module DUWO {
...
+	if not(bel(lowBudget)), not(bel(noBudget))
 			then normalMod.
 	
 	% Use the lowBudget module when our budget is lower than a certain threshold above the target budget
 	if bel(lowBudget), not(bel(noBudget))
 			then lowBudgetMod.
 	
 	% Use the noBudget module when our current budget is beneath our budget target
-	if goal(goalReachBudgetTarget)
+	if bel(lowBudget), bel(noBudget)
```
Comment (Denpeer) [inline src/DUWOagent/DUWO.mod2g:28]: This wasn't necessary

**Comment by MarcianoJ** [comment]
The agent still enters noBudgetMod at the end of each cycle, while it has a high budget

**Comment by Denpeer** [comment]
@MarcianoJ That's because we haven't implemented the fixcycle pr

**Comment by MarcianoJ** [comment]
@Denpeer No, it enters the module just once, but it shouldn't enter the module at all

**Comment by klikooo** [comment]
@MarcianoJ you are right, I will fix it.

**Comment by MarcianoJ** [comment]
I tested it and it works now (with the rule goalBuildStudentHousing :- not(needStudentHousing). since the indicator excel files have been changed)

**Comment by MaxGroenenboom** [comment]
~~Tested and it doesn't work, it freezes in the `normalMod` module.~~
Never mind, it looks like the Tygron servers are offline.

**Comment by klikooo** [comment]
@MaxGroenenboom tygron is down.... check later

**Comment by MaxGroenenboom** [comment]
Looks like it still freezes in normalMod...

File: src/DUWOagent/predicates.pl [hunk 4]
```diff
@@ -18,23 +18,23 @@
...
-		member(Type, Cat).
+	buildings(Y),
+	self(OwnId),
+	member(building(Bid,Name,OwnId,Year,Cat,_,_,_),Y),
+	member(Type, Cat).
 
 % create a predicate with the Id's of all buildable houses of type student, so that the agent can dynamically choose what kind of building to build
 buildableStudentList([]).
 
 buildableStudent(Id,Name) :- functions(FS),
```
Comment (MaxGroenenboom) [inline src/DUWOagent/predicates.pl]: Maybe put `functions(FS)` also on a new line? Also fix the short names in this file as well.

File: src/DUWOagent/actions.act2g [hunk 5]
```diff
@@ -2,34 +2,34 @@ use predicates as knowledge.
 
 % Plan a building action
 define	building_plan_construction(Id, Level, MultiPolygon) with
-   pre{ not(planned(Id, Level, MultiPolygon)) }
-   post {planned(Id, Level, MultiPolygon) }
+   pre { not(planned(Id, Level, MultiPolygon)) }
+   post{planned(Id, Level, MultiPolygon) }
```
Comment (MaxGroenenboom) [inline src/DUWOagent/actions.act2g]: Different indentation than for the other actions' pre and posts?

**Comment by MaxGroenenboom** [comment]
I suppose this can be merged now.

**Comment by klikooo** [comment]
Yes is about time, @MarcianoJ , @Denpeer can you please review?

**Comment by MarcianoJ** [comment]
Approved

**Pull request merged**